napari/utils/misc.py

```diff
@@ -267,6 +267,16 @@ def __str__(self):
         """
         return self.value
 
+    def __eq__(self, other):
+        if type(self) is type(other):
+            return self is other
+        elif isinstance(other, str):
+            return str(self) == other
+        return NotImplemented
```

Comment:
maybe return `False` instead of raising an exception?  if they're not the same type, and not a string... feels very likely they're not equivalent.

Reply:
This doesn't raise a `NotImplementedError`, but returns the special `NotImplemented` value: https://docs.python.org/3/library/constants.html#NotImplemented

One of the reasons I didn't want to override `__eq__` is because Python has some fairly complex (but also reasonable) logic behind `==`. If you override `__eq__` and return `NotImplemented`, you're indicating that you don't know if the LHS and RHS are equal, so it will do some other checks instead (e.g. swapping the LHS and RHS which is why `'additive' == Blending.ADDITIVE` returns True, trying the super class implementation, which for `object` is `self is other`).

I think we could return `False` here for the reason you gave. One possible (though unreasonable) example I thought might break that is:

```python
class TestEnum(StringEnum):
    THING = auto()
    OTHERTHING = auto()

class SuperTestEnum(TestEnum):
    pass

assert TestEnum.THING == SuperTestEnum.THING
```

where it's not even clear to me if those two should be equal. But thankfully, it looks like Python's `EnumMeta._check_for_existing_members` effectively prevents an `Enum` with members from being subclassed.

Therefore, I think returning `False` is probably safe, though `NotImplemented` is also fine. Let me know if you still want me to make the change after reading this.

Reply:
> but returns the special NotImplemented value:

whoops!  sorry, read that too fast :)  I didn't know about the special treatment in the binary methods though!  thanks for pointing that out!  

> I think returning False is probably safe, though NotImplemented is also fine.

yep, I'm convinced.  let's keep it as you have it 👍